app.py

```diff
@@ -72,7 +72,7 @@ def skill():
     Handles Skill requests
     '''
     if request.method == 'GET':
-        return jsonify({})
+        return jsonify(data["skill"])
```

Comment:
The issue also mentions adding tests for the feature. Could you please look into that as well